ui/apps/platform/cypress/integration/vulnmanagement/componentsListPages.test.js

```diff
@@ -31,18 +31,19 @@ describe('Components list Page and its entity detail page, (related entities) su
                 'Nodes',
                 'Risk Priority',
             ]);
+            const pathname = '/main/vulnerability-management/components';
```

Comment:
nit: If this var always means "components path" in this file, maybe it should be called something like `componentsPath`.

And _mutatis mutandis_ for other entity test files.